substrate/utils/frame/benchmarking-cli/src/pallet/writer.rs

```diff
@@ -153,8 +153,8 @@ fn map_results(
 			continue
 		}
 
-		let pallet_string = String::from_utf8(batch.pallet.clone()).unwrap();
-		let instance_string = String::from_utf8(batch.instance.clone()).unwrap();
+		let pallet_name = String::from_utf8(batch.pallet.clone()).unwrap();
+		let instance_name = String::from_utf8(batch.instance.clone()).unwrap();
```

Comment:
Originally i thought that already the types in `BenchmarkBatchSplitResults` and `BenchmarkBatch` should be converted, but it would mean to duplicate or generitize the types.  
Guess this is a good start for now.